index.js

```diff
@@ -57,6 +73,34 @@ router
     ctx.status = 202
     ctx.body = 'OK'
   })
+  .get('/cloudflareError.png', async ctx => {
```

Comment:
Why mark this as async?

Reply:
Idk all other routes do it... just following the pattern

Reply:
I don’t see any obvious reason for it needing to be async and assume that this would actually make the response take slightly longer. Perhaps test if it works fine as a synchronous function?